packages/bsdowl/bsdowl.3.0.0-20150401/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "1.2"
+maintainer: "michipili@gmail.com"
+authors: "Michael Grünewald"
+license: "CeCILL-B"
+homepage: "https://github.com/michipili/bsdowl"
+bug-reports: "https://github.com/michipili/bsdowl/issues"
+dev-repo: "https://github.com/michipili/bsdowl.git"
+tags: [
+  "bsd"
+  "bmake"
+  "build"
+]
+build: [
+  ["./configure" "--prefix" prefix]
+  ["bmake" "-r" "build"] {os != "freebsd"}
```

Comment:
you can simply use `make` here. This is rewritten to `"make"` and `"bmake"` depending on the platform (normally ... if not please report the issue!).